{{cookiecutter.project_slug}}/docs/contributing.md

```diff
@@ -24,8 +24,7 @@ git checkout -b [YOUR FEATURE]
 
 ```sh
 python -m pip install pipx
-python -m pipx install {% if cookiecutter.dependency_management_tool == 'pipenv' -%}pipenv{%- elif cookiecutter.dependency_management_tool == 'poetry' -%}poetry{% endif -%} invoke
-python -m pipx ensurepath
```

Comment:
Do you mean this line is not necessary? But in the documentation of pipx, it seems to be needed if you use it for the first time.

https://github.com/pipxproject/pipx

Reply:
oops, it seems that I misunderstood something. Let me re-walk through the whole process to confirm its dependency. Thanks for the feedback!

Reply:
hmmm I have no luck to install poetry with pipx.

ubuntu-mate 18.04 shows
```
$ pipx --version
0.16.0.0
$ pipx install poetry

No apps associated with package poetry or its dependencies. If you are attempting to install a library,
pipx should not be used. Consider using pip or a similar tool instead.

```

Would you mind showing me your output of `pipx --version` and `pipx install poetry`?

Reply:
hmmm... interesting

This how it looks like when I install it through pipx.

```
$ pipx --version
0.15.5.1
$ pipx install poetry
⚠️  Note: poetry was already on your PATH at ....../.pyenv/shims/poetry
  installed package poetry 1.1.4, Python 3.8.5
  These apps are now globally available
    - doesitcache
    - poetry
    - virtualenv
done! ✨ 🌟 ✨
```

It works even after I upgrade pipx to 0.16.0.0

Reply:
@tai271828 Is there any update on this one?

Reply:
No, QQ. I did not work on this anymore. It's in my wish list of task. Feel free to close the pull request.

Reply:
No worries 🙂 Let's make it a draft one so that I know I don't need to review it at this moment.

Reply:
Sounds good! Draft is a good idea so we could keep the track.

Yeah I have had dug into this issue and found it is a rabbit hole : (